DataAnalytics.md

```diff
@@ -0,0 +1,11 @@
+# Spark on Graviton
+Below are some general guidelines that can be referenced by users trying to improve over all Application performance across Spark clusters. Note that, there is no magic/standard values that works in all use cases and each case can be different. Most of the times, it depends on dataset sizes, cores available, workload type etc. 
+
+1. Shuffle Partitions: This config option is important to achieve optimal performance. Recommendations in regards to this are
```

Comment:
recommendations here, next line "it's recommended"

Reply:
Added perf data showing the delta in next revision